config/locales/kinetic_cafe_error.fr-CA.yml

```diff
@@ -12,3 +12,5 @@ fr-CA:
       header:
         status: Condition
         code: Code
+    cause: >-
+      ^-- causé par: %{message}
```

Comment:
french error logs may be counter-productive, as during a debug session we'll look for "caused by" and will definitely miss "causé par"